### PR TITLE
Update DegreeAv.m

### PR DESCRIPTION
**Pull request by annacanal**
#166

### CODE REVIEW

File: braph/graph/measures/DegreeAv.m [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+    % DegreeAv methods (Static)
+    %   getClass	- returns the average degree class.
+    %   getName     - returns the name of average degree measure.
+    %   getDescription - returns the description of average degree measure.
+    %   is_global	- boolean, checks if average degree measure is global.
+    %   is_nodal    - boolean, checks if average degree measure is nodal.
+    %   is_binodal	- boolean, checks if average degree measure if binodal.
+    %   getMeasure  - returns the average degree class.
+    %   getCompatibleGraphList - returns a list of compatible graphs.
+    %   getCompatibleGraphNumber - returns the number of compatible graphs.
```
Comment (egolol) [inline braph/graph/measures/DegreeAv.m]: align -

File: braph/graph/measures/DegreeAv.m [hunk 2]
```diff
@@ -1,6 +1,40 @@
...
+    %
+    % See also Measure, Degree, Graph, Strength, Distance, Efficency.
+    
     methods
         function m = DegreeAv(g, varargin)
+            % DEGREEAV(G) creates average degree with default measure properties.
+            % G is a graph (e.g, an instance of GraphBD, GraphBU,
+            % GraphWD, Graph WU). 
+            %   
+            % DEGREEAV(G, 'Settings', SETTINGS) ceates average degree measure
```
Comment (egolol) [inline braph/graph/measures/DegreeAv.m]: creates

File: braph/graph/measures/DegreeAv.m [hunk 3]
```diff
@@ -23,35 +61,92 @@
...
+                       
             bool = false;
         end
-        function list = getCompatibleGraphList()  
+        function list = getCompatibleGraphList() 
+            % GETCOMPATIBLEGRAPHLIST returns the list of compatible graphs
+            % to average degree measure
+            %
+            % LIST = GETCOMPATIBLEGRAPHLIST() returns a cell array 
+            % of compatible graph classes to average degree measure.
```
Comment (giovannivolpe) [inline braph/graph/measures/DegreeAv.m]: to average degree

File: braph/graph/measures/DegreeAv.m [hunk 4]
```diff
@@ -23,35 +61,92 @@
...
+            % See also getCompatibleGraphNumber(). 
+            
             list = { ...
                 'GraphBU', ...
                 'GraphWU' ...
                 };
         end
         function n = getCompatibleGraphNumber()
+            % GETCOMPATIBLEGRAPHNUMBER returns the number of compatible
+            % graphs to average degree measure
```
Comment (giovannivolpe) [inline braph/graph/measures/DegreeAv.m]: to average degree

File: braph/graph/measures/DegreeAv.m [hunk 5]
```diff
@@ -23,35 +61,92 @@
...
+            %
+            % N = GETCOMPATIBLEGRAPHNUMBER() returns the number of
+            % compatible graphs to average degree measure.
```
Comment (giovannivolpe) [inline braph/graph/measures/DegreeAv.m]: to average degree

**Pull request merged**